client/src/api/auth.ts

```diff
@@ -1 +1,13 @@
-export {};
+import instance from './axios'
+
+const signInApi = (data: { username: string; password: string }) =>
+  instance
+    .post('/signin', data)
+    .then(result => {
+      console.log('result', result.data)
+    })
+    .catch(error => {
+      console.log('error')
+    })
```

Comment:
result and error handling should be raised up to the caller(the place where we call this function)